src/client-ducky/src/components/SearchBar.js

```diff
@@ -17,41 +59,63 @@ const SearchBar = ({url, method, fullscreen}) => {
     }
 
     function handleClick() {
+        debugger;
         console.log("searched clicked");
+        history.push('/s');
     }
 
     useEffect(() => {
         api.get('parking-sizes/').then(res => {
-            setParkingSpots(res.data)
+            if (res.data.length > 0) {
+                setParkingSpots(res.data)
+            } else {
+                setParkingSpots(fakeData);
+            }
+
         })
     }, [])
 
+    function inputLocationChange(e) {
+        setLocation(e.target.value);
+    }
+
+    function dateInputChange(e) {
+        setDate(e.target.value);
+    }
+
+    function spotTypeInputChange(e) {
+        setSpotType(e.target.value);
+    }
+
+
     return (
         <div className="form-div justify-content-center" style={searchBarStyle}>
             <form  method={method} action={url}>
-                <div className="form-row p-4">
+                <div className="form-row p-3 mx-auto pull-right">
                     <div className="form-group col-lg-4 col-sm-12">
                         <input type="text"
                                className="form-control"
                                placeholder={fullscreen ? "Where are you parking" : "Location"}
                                aria-label="Location"
-                               name="location" />
+                               name="location"
+                                onChange={inputLocationChange}/>
                     </div>
-                    <div className="form-group col-lg-3 col-sm-12">
-                        <input type="date" className="form-control datepicker" placeholder="Select Date" name="date" />
+                    <div className="form-group col-lg-4 col-sm-12">
+                        <input type="date" className="form-control datepicker" placeholder="Select Date" name="date"
+                        onChange={dateInputChange}/>
                     </div>
-                    <div className="form-group col-sm-12 col-lg-4">
-                        <select className="custom-select" id="size-type" name="size-type">
+                    <div className="form-group col-lg-3 col-sm-12">
+                        <select className="custom-select" id="size-type" name="size-type" onChange={spotTypeInputChange}>
                             <option value="" disabled>Parking Spot Size</option>
                             {parkingSpots.map((s) => (
-                                <option name="size-type" key={s.pk} value={s.pk} >
```

Comment:
why did we change this? this is coming from the server's data and it was working...